tests/20-SideMenu/PinnedMenu.test.tsx

```diff
@@ -25,81 +28,216 @@ describe("Boards", () => {
+      });
+    }
   });
   
   test("Correctly marks current board", async () => {
     render(<Boards />);
   
-    //TODO: fill this
+    const currentBoard = screen.getByRole("link", { current: "page" });
+    expect(currentBoard).toHaveAccessibleName(expect.stringContaining("kink-memes"));
+    const svgs = currentBoard.getElementsByTagName("svg");
+    // In the real world it is possible to have the current board be muted, and if we changed the Story to have an example with both at once these length tests would break, but all these tests will break if we change the details of the story boards so I figure it's fine?
+    expect(svgs).toHaveLength(1);
+    expect(svgs[0]).toHaveClass("fa-map-marker-alt");
+    expect(svgs[0]).toBeVisible();
+  });
+  
+  test("Correctly renders board without updates", async () => {
+    render(<Boards />);
+  
+    const oncieBoard = screen.getByRole("link", { name: "oncie-den"}); 
+    expect(within(oncieBoard).queryByRole("presentation", {hidden: true})).not.toBeInTheDocument();
+    const svgs = oncieBoard.getElementsByTagName("svg");
+    expect(svgs).toHaveLength(0);
   });
   
   test("Correctly marks boards with updates", async () => {
     render(<Boards />);
   
-    //TODO: fill this
+    const goreBoard = screen.getByLabelText("gore has new updates");
+    expect(within(goreBoard).getByRole("presentation", {hidden: true})).toBeVisible();
+    expect(within(goreBoard).getByRole("presentation", {hidden: true})).toHaveClass("board-icon__update");
+    const svgs = goreBoard.getElementsByTagName("svg");
+    expect(svgs).toHaveLength(0);
   });
   
   test("Correctly marks outdated boards with updates", async () => {
     render(<Boards />);
   
-    //TODO: fill this
+    const crackBoard = screen.getByLabelText("crack has updates");
+    expect(within(crackBoard).getByRole("presentation", {hidden: true})).toBeVisible();
+    expect(within(crackBoard).getByRole("presentation", {hidden: true})).toHaveClass("board-icon__update");
+    const svgs = crackBoard.getElementsByTagName("svg");
+    expect(svgs).toHaveLength(0);
   });
   
   test("Correctly marks muted board", async () => {
     render(<Boards />);
   
-    //TODO: fill this
+    const mutedBoard = screen.getByLabelText("anime muted");
+    expect(screen.queryByLabelText("anime has updates")).not.toBeInTheDocument();
+    expect(screen.queryByLabelText("anime has new updates")).not.toBeInTheDocument();
+    expect(within(mutedBoard).queryByRole("presentation", {hidden: true})).not.toBeInTheDocument();
+    const svgs = mutedBoard.getElementsByTagName("svg");
+    expect(svgs).toHaveLength(1);
+    expect(svgs[0]).toHaveClass("fa-volume-mute");
+    expect(svgs[0]).toBeVisible();
   });
 });
 
 describe("Loading", () => {
   test("Renders loading section", async () => {
     render(<Loading />);
   
-    //TODO: fill this
+    expect(screen.getByLabelText(`loading ${(Loading.args as PinnedMenuSectionProps).sectionId}`)).toBeVisible();
```

Comment:
I don't have an answer to this (I might venture into finding an accessibility expert), but I see that the label text for boards is an active description ("anime has updates") vs a description of the item ("loading pinned item placeholder").

I don't know what the right way of doing this would be, but I think "pinned items are loading"/"pinned item is loading" would work to ensure consistency.

Reply:
My intuition is that labels should be a description of the item unless there's a pressing need for them to be otherwise, like with the "anime has updates" (I thought really hard about what the best screen-reader experience would be for the board menu and settled on "board has (new) updates" as the best balance of concise and not confusing if you're being read a whole list of them), I don't know that I'm actually right about that though.

If you think consistency is more important for now until we have an actual answer, I can change it.

Reply:
Yeah let's keep it consistent for now and try to think about what's a good experience in terms of hearing the labels spoken. We might want to invest some time in finding good guidance for labels, but let's not punt on this right now since there's a lot of stuff we have to fix before this is an acceptable level of accessible anyway.

Reply:
Okay. Should I go and change the labels on the loading boards placeholders as well?

Reply:
Went ahead and changed them